### ISSUE TITLE
Chat Panel wrong initial aria message

### ISSUE THREAD

**Issue by rebornix** [microsoft/vscode#186400]
Testing #186218

When focused on the chat panel the first time, NVDA read `Chat input: type code and press enter to run`. I don't think it's correct to call it "type code", it should be mentioning chat or conversation.

**Comment by meganrogge** [comment]
This instruction is meant to explicitly describe what action can be taken. What would you suggest it say instead?

**Comment by rebornix** [comment]
> This instruction is meant to explicitly describe what action can be taken

@meganrogge but isn't `insert code` wrong? 

> What would you suggest it say instead

We can at least align with place holder: "Type to ask Copilot questions or type / for topics, press enter to send out the request"

**Comment by meganrogge** [comment]
/duplicate https://github.com/microsoft/vscode-copilot/issues/334
